lib/engines/hckinstall/iso.json

```diff
@@ -1,5 +1,5 @@
 {
-  "Win2019x64": {
+  "Win2022x64_bios_CORE": {
```

Comment:
- It's 2019, not 2022.
- I think the `_CORE` suffix should be removed for consistency with the names of platforms that use it.

Reply:
in any case, this is just an example without real `product_key` this file does not work)